definitions/family-private/json/AuthorisationCaseField.json

```diff
@@ -289,140 +289,280 @@
     "CRUD": "R"
   },
   {
     "LiveFrom": "13/08/2021",
     "CaseTypeID": "C100",
-    "CaseFieldID": "AssistanceRequired",
+    "CaseFieldID": "LabelSpecialArrangements",
+    "UserRole": "caseworker-privatelaw-solicitor",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangements",
+    "UserRole": "caseworker-privatelaw-judge",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangements",
+    "UserRole": "caseworker-privatelaw-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangements",
+    "UserRole": "caseworker-privatelaw-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangementsDescription",
+    "UserRole": "caseworker-privatelaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangementsDescription",
+    "UserRole": "caseworker-privatelaw-solicitor",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangementsDescription",
+    "UserRole": "caseworker-privatelaw-judge",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangementsDescription",
+    "UserRole": "caseworker-privatelaw-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangementsDescription",
+    "UserRole": "caseworker-privatelaw-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "IsSpecialArrangementsRequired",
+    "UserRole": "caseworker-privatelaw-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "IsSpecialArrangementsRequired",
     "UserRole": "caseworker-privatelaw-solicitor",
     "CRUD": "CRU"
   },
   {
     "LiveFrom": "13/08/2021",
     "CaseTypeID": "C100",
-    "CaseFieldID": "AssistanceRequired",
+    "CaseFieldID": "IsSpecialArrangementsRequired",
+    "UserRole": "caseworker-privatelaw-judge",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "IsSpecialArrangementsRequired",
+    "UserRole": "caseworker-privatelaw-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "IsSpecialArrangementsRequired",
+    "UserRole": "caseworker-privatelaw-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "13/08/2021",
+    "CaseTypeID": "C100",
+    "CaseFieldID": "LabelSpecialArrangementsRequired",
+    "UserRole": "caseworker-privatelaw-courtadmin",
+    "CRUD": "R"
```

Comment:
Labels don't need authorisation - this can be removed.

Reply:
They need one when we want to display them in a tab

Reply:
I've removed them for now as we haven't done the tab as part of this story :)